style.css

```diff
@@ -0,0 +1,70 @@
+body {
+  display: flex;
+  justify-content: center;
+}
+
+header {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 35%;
```

Comment:
As asked by the previous reviewer you can't fix the width as you did. Please correct it.

See what's happen when I open your project
<img width="723" alt="image" src="https://user-images.githubusercontent.com/7243132/170838596-4b940965-df86-4ec8-839c-a44a7e9d5cd1.png">